websites/H/Homestuck/presence.ts

```diff
@@ -0,0 +1,310 @@
+const presence = new Presence({
+		clientId: "941798064694378557"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			startTimestamp: browsingTimestamp
+		},
+		pathStr = document.location.pathname,
```

Comment:
You will have to update all matches too 
```suggestion
		{ pathname } = document.location,
```